apps/fluent-tester/src/E2E/Menu/specs/Menu.spec.win.ts

```diff
@@ -0,0 +1,121 @@
+  it('Wait for app load', () => {
+    NavigateAppPage.waitForPageDisplayed(BOOT_APP_TIMEOUT);
+    expect(NavigateAppPage.isPageLoaded()).toBeTruthy(NavigateAppPage.ERRORMESSAGE_APPLOAD);
+  });
+
+  it('Click and navigate to Button test page', () => {
+    /* Scroll to component test page button in scrollview if not already visible*/
+    MenuPageObject.scrollToComponentButton();
+    MenuPageObject.waitForButtonDisplayed(PAGE_TIMEOUT);
+
+    /* Click on component button to navigate to test page */
+    NavigateAppPage.clickAndGoToMenuPage();
+    MenuPageObject.waitForPageDisplayed(PAGE_TIMEOUT);
+
+    expect(MenuPageObject.isPageLoaded()).toBeTruthy(MenuPageObject.ERRORMESSAGE_PAGELOAD);
+    expect(MenuPageObject.didAssertPopup()).toBeFalsy(MenuPageObject.ERRORMESSAGE_ASSERT); // Ensure no asserts popped up
+  });
+});
+
+describe('Experimental Button Functional Testing', () => {
+  /* Scrolls and waits for the Button to be visible on the Test Page */
+  beforeEach(() => {
+    MenuPageObject.scrollToTestElement();
+    MenuPageObject.waitForPrimaryElementDisplayed(PAGE_TIMEOUT);
+  });
+
+  it('Validate OnOpenChange() callback was fired -> Click', () => {
+    MenuPageObject.clickComponent();
+    expect(MenuPageObject.didMenuOpen()).toBeTruthy();
+    expect(MenuPageObject.didAssertPopup()).toBeFalsy(MenuPageObject.ERRORMESSAGE_ASSERT);
+
+    MenuPageObject.clickComponent();
+    expect(MenuPageObject.didMenuClose()).toBeTruthy();
+    expect(MenuPageObject.didAssertPopup()).toBeFalsy(MenuPageObject.ERRORMESSAGE_ASSERT);
+  });
+
+  it('Validate OnOpenChange() callback was fired -> Type "Enter"', () => {
+    MenuPageObject.sendKey(MenuComponentSelector.PrimaryComponent, Keys.Enter);
+    expect(MenuPageObject.didMenuOpen()).toBeTruthy();
+    expect(MenuPageObject.didAssertPopup()).toBeFalsy(MenuPageObject.ERRORMESSAGE_ASSERT);
+
+    MenuPageObject.sendKey(MenuComponentSelector.PrimaryComponent, Keys.Enter);
+    expect(MenuPageObject.didMenuClose()).toBeTruthy();
+    expect(MenuPageObject.didAssertPopup()).toBeFalsy(MenuPageObject.ERRORMESSAGE_ASSERT);
+  });
+
+  it('Validate OnOpenChange() callback was fired -> Type "Spacebar"', () => {
+    MenuPageObject.sendKey(MenuComponentSelector.PrimaryComponent, Keys.Spacebar);
+    expect(MenuPageObject.didMenuOpen()).toBeTruthy();
+    expect(MenuPageObject.didAssertPopup()).toBeFalsy(MenuPageObject.ERRORMESSAGE_ASSERT);
+
+    MenuPageObject.sendKey(MenuComponentSelector.PrimaryComponent, Keys.Spacebar);
+    expect(MenuPageObject.didMenuClose()).toBeTruthy();
+    expect(MenuPageObject.didAssertPopup()).toBeFalsy(MenuPageObject.ERRORMESSAGE_ASSERT);
+  });
+});
+
+describe('Experimental Button Accessibility Testing', () => {
+  it('Experimental Button - Validate accessibilityRole of menu item is correct', () => {
+    MenuPageObject.scrollToTestElement();
+    MenuPageObject.waitForPrimaryElementDisplayed(PAGE_TIMEOUT);
```

Comment:
It doesn't refer to this PR, it's just an idea. Maybe it would be good to have setup and tearDown methods for the tests. I see in a lot of our E2E tests there are repeating parts. Maybe it would be good in future to move it to setup part.